library/foundation/src/main/java/com/epmedu/animeal/foundation/dialog/AnimealAlertDialog.kt

```diff
@@ -0,0 +1,81 @@
+package com.epmedu.animeal.foundation.dialog
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.foundation.button.AnimealButton
+import com.epmedu.animeal.foundation.theme.AnimealTheme
+import com.epmedu.animeal.foundation.theme.DisabledButtonColor
+
+@Composable
+fun AnimealAlertDialog(
+    title: String,
+    dismissText: String,
+    acceptText: String,
+    onDismiss: () -> Unit = {},
+    onConfirm: () -> Unit = {},
+    content: @Composable (() -> Unit)? = null,
+) {
+    AlertDialog(
+        modifier = Modifier.padding(24.dp),
+        shape = RoundedCornerShape(32.dp),
+        onDismissRequest = {
+            // showAlert.value = false
```

Comment:
You probably forgot to call onDismiss callback here

Reply:
Updated